.github/CODEOWNERS

```diff
@@ -422,6 +422,8 @@
 /core/tests/ @nithusha21
 /core/tests/protractor_utils @U8NWXD
 /core/tests/protractor_desktop @U8NWXD
+/core/tests/protractor @U8NWXD
```

Comment:
End this with  '/' if its a directory (Ditto for the above cases)

Reply:
Done